### ISSUE TITLE
Buttons do not have an accessible name

### ISSUE THREAD

**Issue by stigsfoot** [nditech/demgames-marketing-public#4]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

**Affected**
`button.navbar-toggler.hidden-lg-up.float-lg-right`
